docs/contributing/security.md

```diff
@@ -41,7 +41,7 @@ On the day of disclosure, we will take the following steps:
    The commit messages for these patches will indicate that they are for security issues, but will not describe the issue in any detail; instead, they will warn of upcoming disclosure.
 2. Issue the relevant release(s), by placing new packages on [the Python Package Index](https://pypi.org/project/wagtail/), tagging the new release(s) in Wagtail\'s GitHub repository and updating Wagtail\'s [release notes](../releases/index).
 3. Post a public entry on [Wagtail\'s blog](https://wagtail.org/blog/), describing the issue and its resolution in detail, pointing to the relevant patches and new releases, and crediting the reporter of the issue (if the reporter wishes to be publicly identified).
-4. Post a notice to the [Wagtail support forum](https://groups.google.com/d/forum/wagtail) and Twitter feed ([\@WagtailCMS](https://twitter.com/wagtailcms)) that links to the blog post.
+4. Post a notice to the [Wagtail support forum](https://groups.google.com/g/wagtail) and Twitter feed ([\@WagtailCMS](https://twitter.com/wagtailcms)) that links to the blog post.
```

Comment:
question: are we still actively encouraging people to use Google Groups? My impression was we switched to GitHub Discussions

Reply:
I thought you would ask this haha. I thought the same but it looks like Matt is still posting to it.

<img width="1436" alt="Screen Shot 2022-10-04 at 8 48 01 pm" src="https://user-images.githubusercontent.com/1396140/193800606-660d659a-9df3-4c93-a5c5-a908e7d612c4.png">

I think we should update the link but probably a good question for the core team - I feel like we have discussed this before but cannot remember the outcome.